infrastructure/environments/prod/main.tf

```diff
@@ -39,13 +39,14 @@ resource "azurerm_resource_group" "keyvault" {
 module "products" {
   source = "../../modules/products"
 
-  environment         = var.ENVIRONMENT
-  location            = var.REGION
-  namespace           = local.namespace
-  pars_namespace      = local.pars_namespace
-  resource_group_name = data.azurerm_resource_group.products.name
-  search_sku          = "standard"
-  app_registration_owners = var.KEYVAULT_AUTHORISED_PERSON_IDS
+  environment                       = var.ENVIRONMENT
+  location                          = var.REGION
+  namespace                         = local.namespace
+  pars_namespace                    = local.pars_namespace
+  resource_group_name               = data.azurerm_resource_group.products.name
+  search_sku                        = "standard"
+  app_registration_owners           = var.KEYVAULT_AUTHORISED_PERSON_IDS
+  addtional_allowed_pars_reply_urls = ["https://pars.mhra.gov.uk"]
```

Comment:
assuming this is the dns name we'll get ?

Reply:
should we allow CDN and static container ones too?

Reply:
they are, but they are in the module directly

Reply:
I see them, `azurerm_cdn_endpoint.pars.host_name` and `azurerm_storage_account.pars.primary_web_host`